test/core/end2end/bad_server_response_test.c

```diff
@@ -82,7 +82,7 @@
 #define HTTP1_DETAIL_MSG "Trying to connect an http1.x server"
 
 /* TODO(zyc) Check the content of incomming data instead of using this length */
-#define EXPECTED_INCOMING_DATA_LENGTH (size_t)310
+#define EXPECTED_INCOMING_DATA_LENGTH (size_t)300
```

Comment:
@y-zeng can you shed some light on the role this plays in the test? The naming is a bit misleading currently.